native/src/transportRouteResult.h

```diff
@@ -0,0 +1,116 @@
+#ifndef _OSMAND_TRANSPORT_ROUTE_RESULT_H
+#define _OSMAND_TRANSPORT_ROUTE_RESULT_H
+#include "transportRouteResultSegment.h"
+#include "transportRoutingConfiguration.h"
+#include "transportRoutingContext.h"
+ 
+
+struct TransportRouteResult {
+    vector<SHARED_PTR<TransportRouteResultSegment>> segments;
+    double finishWalkDist;
+    double routeTime;
+    SHARED_PTR<TransportRoutingConfiguration> config;
+
+    TransportRouteResult(TransportRoutingContext* ctx) {
```

Comment:
ctx -reference ??

Reply:
TransportRoutingConfiguration, TransportRoutingContext should not be shared_ptr

Reply:
vector<SHARED_PTR<TransportRouteResultSegment>> segments; - shared_ptr ? should be vector<TransportRouteResultSegment>